schema/me/conversation.js

```diff
@@ -74,6 +77,28 @@ export const ConversationParticipantType = new GraphQLObjectType({
   },
 })
 
+export const ConversationResponderType = new GraphQLObjectType({
```

Comment:
I was thinking about that but it felt a bit weird because it would have two optional fields - `email` and `reply_to_impulse_ids`. The conversation recipient will never have an `email` and the conversation initiator will never have a `reply_to_impulse_ids`.